Sources/ConsoleKit/Terminal/ANSI.swift

```diff
@@ -1,7 +1,15 @@
-import Foundation
-
-#if canImport(Android)
+#if os(macOS)
```

Comment:
this should probably be `#if canImport(Darwin)`.
I'd try to refer to SwiftNIO in how to import these different modules. Maybe some stuff won't apply to this project, but i think most stuff do apply here to.

Reply:
example from NIO: https://github.com/apple/swift-nio/blob/37094b1b424d01ee9ad1b9188a35ab1204ef1db9/Sources/NIOFileSystem/Internal/SystemFileHandle.swift#L20

I'd just search e.g. `canImport(Darwin)` and try to work out if we should do what they've done in those places, which most of the times should be a yes.